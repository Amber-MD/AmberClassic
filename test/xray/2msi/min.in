minimisation stage-1
 &cntrl
  imin=1, maxcyc=500, ncyc=50,
  ntb=1,
  ntr=0,
  cut=8.,
 /
