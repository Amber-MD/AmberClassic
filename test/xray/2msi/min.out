
          -------------------------------------------------------
          Amber 20 PMEMD                              2020
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 18

|  Compiled date/time: Fri Dec 10 09:15:18 2021
| Run on 01/03/2022 at 09:09:04

|   Executable path: /home/case/sizmailov/bin/pmemd.cuda_SPFP
| Working directory: /home/case/msander/test/xray/2msi
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|   MDIN: min.in                                                                
|  MDOUT: min.out                                                               
| INPCRD: wbox.inpcrd                                                           
|   PARM: wbox.prmtop                                                           
| RESTRT: min.rst                                                               
|   REFC: refc                                                                  
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: mdcrd                                                                 
| MDINFO: mdinfo                                                                
|  MDFRC: mdfrc                                                                 


 Here is the input file:

minimisation stage-1                                                           
 &cntrl                                                                        
  imin=1, maxcyc=500, ncyc=50,                                                 
  ntb=1,                                                                       
  ntr=0,                                                                       
  cut=8.,                                                                      
 /                                                                             


Note: ig = -1. Setting random seed to   885578 based on wallclock time in 
      microseconds.
| irandom = 1, using AMBER's internal random number generator (default).
 
|--------------------- INFORMATION ----------------------
| GPU (CUDA) Version of PMEMD in use: NVIDIA GPU IN USE.
|                    Version 18.0.0
| 
|                      03/25/2018
| 
| Implementation by:
|                    Ross C. Walker     (SDSC)
|                    Scott Le Grand     (nVIDIA)
| 
| Version 18 performance extensions by:
|                    David Cerutti     (Rutgers)
| 
| Precision model in use:
|      [SPFP] - Single Precision Forces, 64-bit Fixed Point
|               Accumulation. (Default)
| 
|--------------------------------------------------------
 
|----------------- CITATION INFORMATION -----------------
|
|    When publishing work that utilized the CUDA version
|    of AMBER, please cite the following in addition to
|    the regular AMBER citations:
|
|  - Romelia Salomon-Ferrer; Andreas W. Goetz; Duncan
|    Poole; Scott Le Grand; Ross C. Walker "Routine
|    microsecond molecular dynamics simulations with
|    AMBER - Part II: Particle Mesh Ewald", J. Chem.
|    Theory Comput., 2013, 9 (9), pp3878-3888,
|    DOI: 10.1021/ct400314y.
|
|  - Andreas W. Goetz; Mark J. Williamson; Dong Xu;
|    Duncan Poole; Scott Le Grand; Ross C. Walker
|    "Routine microsecond molecular dynamics simulations
|    with AMBER - Part I: Generalized Born", J. Chem.
|    Theory Comput., 2012, 8 (5), pp1542-1555.
|
|  - Scott Le Grand; Andreas W. Goetz; Ross C. Walker
|    "SPFP: Speed without compromise - a mixed precision
|    model for GPU accelerated molecular dynamics
|    simulations.", Comp. Phys. Comm., 2013, 184
|    pp374-380, DOI: 10.1016/j.cpc.2012.09.022
|
|--------------------------------------------------------
 
|------------------- GPU DEVICE INFO --------------------
|
|            CUDA_VISIBLE_DEVICES: 0
|   CUDA Capable Devices Detected:      1
|           CUDA Device ID in use:      0
|                CUDA Device Name: Tesla V100-PCIE-16GB
|     CUDA Device Global Mem Size:  16160 MB
| CUDA Device Num Multiprocessors:     80
|           CUDA Device Core Freq:   1.38 GHz
|
|--------------------------------------------------------
 
 
| Conditional Compilation Defines Used:
| FFTW_FFT
| BINTRAJ
| MKL
| CUDA
| EMIL

| Largest sphere to fit in unit cell has radius =    16.635

| New format PARM file being parsed.
| Version =    1.000 Date = 10/26/21 Time = 13:28:52

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 getting new box info from bottom of inpcrd
 NATOM  =    6499 NTYPES =      14 NBONH =    4547 MBONA  =    1976
 NTHETH =    4768 MTHETA =    2700 NPHIH =    9192 MPHIA  =    8976
 NHPARM =       0 NPARM  =       0 NNB   =   25660 NRES   =    1081
 NBONA  =    1976 NTHETA =    2700 NPHIA =    8976 NUMBND =      49
 NUMANG =     115 NPTRA  =     149 NATYP =      23 NPHB   =       1
 IFBOX  =       1 NMXRS  =      24 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

| Coordinate Index Table dimensions:     7    8    9
| Direct force subcell size =     4.7529    4.9863    4.9489

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =      50, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =     500, ncyc    =      50, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   33.270   Box Y =   39.890   Box Z =   44.540
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   36       NFFT2 =   40       NFFT3 =   48
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:      817

     Sum of charges from parm topology file =  -0.00000012
     Forcing neutrality...

| Dynamic Memory, Types Used:
| Reals              267843
| Integers           625661

| Nonbonded Pairs Initial Allocation:     1085495

| GPU memory information (estimate):
| KB of GPU memory in use:     33933
| KB of CPU memory in use:     21333

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       3.1745E+09     4.4175E+02     1.9463E+04     CE       2038

 BOND    =      232.8299  ANGLE   =      780.1303  DIHED      =     3054.0493
 VDWAALS = *************  EEL     =   -19989.2548  HBOND      =        0.0000
 1-4 VDW =     1168.3234  1-4 EEL =    10216.0469  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -9.6319E+03     1.0458E+01     2.6912E+02     H1       6447

 BOND    =      817.3616  ANGLE   =      706.3791  DIHED      =     3084.6715
 VDWAALS =      104.1245  EEL     =   -25241.2914  HBOND      =        0.0000
 1-4 VDW =      975.2992  1-4 EEL =     9921.5669  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -1.5125E+04     2.8990E+00     4.3634E+01     O        5114

 BOND    =      818.9821  ANGLE   =      507.2028  DIHED      =     3090.9520
 VDWAALS =    -1346.0308  EEL     =   -28695.9367  HBOND      =        0.0000
 1-4 VDW =      834.0561  1-4 EEL =     9665.9684  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    150      -1.6272E+04     2.2067E+00     5.0582E+01     CD       1557

 BOND    =      843.2093  ANGLE   =      504.7672  DIHED      =     3089.0175
 VDWAALS =    -1388.7774  EEL     =   -29742.0579  HBOND      =        0.0000
 1-4 VDW =      807.4161  1-4 EEL =     9614.5677  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -1.6841E+04     1.7342E+00     5.8886E+01     C        1010

 BOND    =      846.5302  ANGLE   =      498.1891  DIHED      =     3081.4815
 VDWAALS =    -1305.6332  EEL     =   -30352.6014  HBOND      =        0.0000
 1-4 VDW =      799.0537  1-4 EEL =     9591.6117  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    250      -1.7196E+04     1.3490E+00     3.0549E+01     CG        904

 BOND    =      874.0044  ANGLE   =      495.8914  DIHED      =     3075.2619
 VDWAALS =    -1239.5631  EEL     =   -30768.1334  HBOND      =        0.0000
 1-4 VDW =      792.3136  1-4 EEL =     9574.3056  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300      -1.7428E+04     1.2001E+00     3.5493E+01     CG        904

 BOND    =      892.4772  ANGLE   =      492.5075  DIHED      =     3068.6362
 VDWAALS =    -1183.7711  EEL     =   -31050.7652  HBOND      =        0.0000
 1-4 VDW =      788.4412  1-4 EEL =     9564.5926  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    350      -1.7615E+04     9.9206E-01     3.6110E+01     CG        904

 BOND    =      907.7970  ANGLE   =      491.7781  DIHED      =     3062.8755
 VDWAALS =    -1126.6302  EEL     =   -31295.9082  HBOND      =        0.0000
 1-4 VDW =      786.4432  1-4 EEL =     9558.3017  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -1.7772E+04     8.2472E-01     1.5620E+01     CG        904

 BOND    =      924.4978  ANGLE   =      491.9491  DIHED      =     3058.7373
 VDWAALS =    -1095.2760  EEL     =   -31486.6492  HBOND      =        0.0000
 1-4 VDW =      782.7228  1-4 EEL =     9552.4699  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    450      -1.7960E+04     1.1903E+00     3.8817E+01     CG        904

 BOND    =      940.3044  ANGLE   =      491.2047  DIHED      =     3054.5294
 VDWAALS =    -1029.6255  EEL     =   -31748.9636  HBOND      =        0.0000
 1-4 VDW =      782.5340  1-4 EEL =     9550.4874  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -1.8061E+04     7.9122E-01     2.1660E+01     CD       2569

 BOND    =      950.9278  ANGLE   =      490.1925  DIHED      =     3052.0487
 VDWAALS =     -999.7499  EEL     =   -31883.6777  HBOND      =        0.0000
 1-4 VDW =      780.9198  1-4 EEL =     9548.5248  RESTRAINT  =        0.0000


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -1.8061E+04     7.9122E-01     2.1660E+01     CD       2569

 BOND    =      950.9278  ANGLE   =      490.1925  DIHED      =     3052.0487
 VDWAALS =     -999.7499  EEL     =   -31883.6777  HBOND      =        0.0000
 1-4 VDW =      780.9198  1-4 EEL =     9548.5248  RESTRAINT  =        0.0000
--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|  NonSetup CPU Time in Major Routines:
|
|     Routine           Sec        %
|     ------------------------------
|     Nonbond           0.08   29.69
|     Bond              0.00    0.00
|     Angle             0.00    0.00
|     Dihedral          0.00    0.00
|     Shake             0.00    0.00
|     Other             0.18   70.31
|     ------------------------------
|     Total             0.26

|  PME Nonbond Pairlist CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     Set Up Cit           0.00    0.00
|     Build List           0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  PME Direct Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     NonBonded Calc       0.00    0.00
|     Exclude Masked       0.00    0.00
|     Other                0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  PME Reciprocal Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     1D bspline           0.00    0.00
|     Grid Charges         0.00    0.00
|     Scalar Sum           0.00    0.00
|     Gradient Sum         0.00    0.00
|     FFT                  0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  Setup CPU time:            0.90 seconds
|  NonSetup CPU time:         0.26 seconds
|  Total CPU time:            1.16 seconds     0.00 hours

|  Setup wall time:           1    seconds
|  NonSetup wall time:        0    seconds
|  Total wall time:           1    seconds     0.00 hours
